encrypted-media-respec.html

```diff
@@ -2690,9 +2694,10 @@
         <p><em>All</em> data associated with a session MUST be cleared when the session is cleared, such as in <a def-id="update"></a> when processing a license destruction acknowledgement or key usage record acknowledgement.
           See <a href="#persistent-state-requirements">Persistent Data</a>.
         </p>
-        <p>The CDM MUST ensure that data for a given session is only present in one active unclosed session in any <a def-id="document-concept"></a>.
+        <p>The CDM MUST ensure that data for a given session is only present in one <a>MediaKeySession</a> object in any <a def-id="document-concept"></a> that is not
```

Comment:
"that" near the end of this line refers to the Document, which is not the intent.